### PR TITLE
Support encrypting of pgautofailover enterprise

### PR DESCRIPTION
**Pull request by JelteF**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,23 +12,47 @@ env:
...
+    # Packages on packages.microsoft.com
     - TARGET_PLATFORM=el/6
-    - TARGET_PLATFORM=ol/7
-    - TARGET_PLATFORM=ol/6
+    # Removed for now because it has no gcc4.8+ (which is required for security
+    # compliance)
+    # - TARGET_PLATFORM=ol/6
+    - TARGET_PLATFORM=el/7
+    # Removed for now since we don't have oracle linux repos on microsoft
+    # - TARGET_PLATFORM=ol/7
```
Comment (hanefi) [inline .travis.yml:22]: I really like the comments that state the reason why you intentionally left out some distros

File: .travis.yml [hunk 2]
```diff
@@ -12,23 +12,47 @@ env:
...
+    - TARGET_PLATFORM=el/8
     - TARGET_PLATFORM=debian/buster
-    - TARGET_PLATFORM=debian/jessie
     - TARGET_PLATFORM=debian/stretch
+    - TARGET_PLATFORM=debian/jessie
     - TARGET_PLATFORM=ubuntu/bionic
     - TARGET_PLATFORM=ubuntu/xenial
```
Comment (hanefi) [inline .travis.yml:28]: `ubuntu/focal` and `ol/8` are missing on both packagecloud and microsoft targets. Is that intentional?

I guess we'll not have the `ol/8` on MS repositories, however for consistency purposes we can also add this to the comments there maybe.

File: debian/decrypt-files.sh [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+interactive_license=false
+while [ -z "$CITUS_ACCEPT_LICENSE" ]; do
+    interactive_license=true
+    echo "Do you accept these terms? YES/NO"
+    read -r CITUS_ACCEPT_LICENSE
+done
+
+case "$CITUS_ACCEPT_LICENSE" in
+    YES );;
+    y|Y|Yes|yes )
```
Comment (hanefi) [inline debian/decrypt-files.sh:38]: This seems really unintuitive to me. I am used to typing a single letter (y or n) when such a prompt appears without actually reading all my options.

If there is a solid reason thy only `YES` is allowed, I'm totally ok with this

Comment (JelteF) [reply debian/decrypt-files.sh:38]: Legel team prefered it that way. (It's the same for the citus enterprise microsoft packages.)

File: .travis.yml (hunk 1 above)
Comment (JelteF) [reply .travis.yml:22]: Resolved in https://github.com/citusdata/packaging/pull/473

File: .travis.yml (hunk 2 above)
Comment (JelteF) [reply .travis.yml:28]: Resolved in https://github.com/citusdata/packaging/pull/473

**Comment by JelteF** [comment]
Closing this in favor of Resolved in https://github.com/citusdata/packaging/pull/473

**Pull request closed without merging**